UOP1_Project/Assets/Scripts/CameraManager.cs

```diff
@@ -96,4 +128,23 @@ private void OnFrameObjectEvent(Transform value)
 	{
 		SetupProtagonistVirtualCamera(value);
 	}
+
+	private void OnCameraSwapEvent(CinemachineVirtualCamera vcam)
+    {
+		// TODO: find a better way than +10 , -10 priority. may break if scene is setup incorrectly
+        if(vcam != null){
+			if(!cinemachineBrain.IsLive(vcam)){
+				cinemachineBrain.ActiveVirtualCamera.Priority -= 10;
+				vcam.Priority += 10;
```

Comment:
Isn't it safer to use absolute numbers? Like 100 and 0.
Feels safer in case for some reason this function is called twice (-20, +20), then you call the exit once  and it's not enough to bring the situation back to normal.

Reply:
implemented